### ISSUE TITLE
Make it optional

### ISSUE THREAD

**Issue by SubatomicPlanets** [undreamai/LLMUnity#169]
### Describe the feature

LLMs take up a lot of storage space and need powerful devices to run on. If it were somehow optional, so that you could download everything you need (like the model, DLLs, and so on) and put it in a specific location on the device, it would be really cool.

I am making a game for Windows, Android, and macOS. I have it highly optimized, and it also works on very old devices and takes up only around 100 megabytes of storage space. The problem is that with LLM for Unity, it goes up in storage size. 

I managed to make the model and the llamafile optional, But in 2.0.0, the llamafile is replaced by DLLs. I don't know if you could somehow detect if they are present, and if not, then don't try loading them. I don't know how DLLs work or if you can make them optional like that...

The dlls on Windows use around 70 megabytes. That is nearly half my game size. I would prefer not to waste so much space when the user does not want to use LLMs and has not downloaded the model.

Basically, what I need is this:
- When the user starts the game, it checks if the correct DLLs and models are present
- If they are, it loads them and continues as usual
- If not, then chat requests won't work (you could also add a variable like "LLMCanRun" or so to use in custom scripts)

I don't know exactly how the user would install this "LLM pack"  but maybe it could be a simple folder that you put in the Application.persistentDataPath or something like that.

Again, I don't know if this can work, but I basically need it to work... Just let me know if it doesn't work for some reason. 
Thanks!

**Comment by amakropoulos** [comment]
I added this functionality.
Now if the DLLs do not exist:
- during the server start, there will be a log error 
- when a method e.g. chat is called, there will be a log error and an exception thrown
